### PR TITLE
lpd8editor: init at v0.0.16

### PR DESCRIPTION
**Pull request by pinpox**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/lpd8editor/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ lib
+, qt5
+, stdenv
+, git
+, fetchFromGitHub
+, cmake
+, alsaLib
```
[the remark this answers is not in the document]
Comment (pinpox) [reply pkgs/applications/audio/lpd8editor/default.nix]: Thanks! Should be fixed now.

**Comment by pinpox** [comment]
@ofborg build lpd8editor

File: pkgs/applications/audio/lpd8editor/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  nativeBuildInputs = [
+    cmake
+    alsa-lib
+    git
+    qt5.wrapQtAppsHook
+  ];
+
+  meta = with lib; {
+    description = "A linux editor for the Akai LPD8";
+    homepage = "git@github.com:charlesfleche/lpd8editor.git";
```
Comment (infinisil) [inline pkgs/applications/audio/lpd8editor/default.nix:33]: Should be an https URL

File: pkgs/applications/audio/lpd8editor/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    repo = "lpd8editor";
+    rev = "v${version}";
+    hash = "sha256-lRp2RhNiIf1VrryfKqYFSbKG3pktw3M7B49fXVoj+C8=";
+  };
+
+  buildInputs = [ qttools ];
+
+  nativeBuildInputs = [
+    cmake
+    alsa-lib
```
Comment (infinisil) [inline pkgs/applications/audio/lpd8editor/default.nix:26]: `alsa-lib` should be in `buildInputs` (you can also enable `strictDeps = true` to check for this)

**Review by infinisil — CHANGES_REQUESTED** [review]
Sorry I'm busy with a lot of things, but here's a quick review. Otherwise it looks good to me :)

**Comment by pinpox** [comment]
Applied the suggested changes and re-checked that it still builds.

**Review by infinisil — APPROVED** [review]

**Pull request merged**